content/en/network_monitoring/devices/device_topology_map.md

```diff
@@ -27,7 +27,7 @@ The Datadog Agent version 7.52 and later automatically collects topology data. N
 
 ### Prerequisites
 
-1. Devices have LLDP (Link Layer Discovery Protocol) and/or CDP (Cisco Discovery Protocol) enabled with SNMP.
+1. Devices have LLDP (Link Layer Discovery Protocol) and/or CDP (Cisco Discovery Protocol) enabled with SNMP. The same protocol should be used on any two connected devices for them to discover each other. LLDP is generally preferred as it is a more common option.
```

Comment:
```suggestion
1. Devices have LLDP (Link Layer Discovery Protocol) and/or CDP (Cisco Discovery Protocol) enabled with SNMP. Use the same protocol on connected devices so that they can discover each other. LLDP is generally preferred as it is a more common option.
```